osu.Game.Tests/Visual/SongSelectV2/TestSceneBeatmapCarouselArtistGrouping.cs

```diff
@@ -19,7 +18,9 @@ public void SetUpSteps()
         {
             RemoveAllBeatmaps();
             CreateCarousel();
-            SortBy(new FilterCriteria { Group = GroupMode.Artist, Sort = SortMode.Artist });
+
+            SortBy(SortMode.Artist);
+            GroupBy(GroupMode.Artist);
```

Comment:
This seems worse than what we had 🤷

Reply:
https://github.com/ppy/osu/pull/33046/commits/8775731c2442416015148c5984c312fd173c6849